skymap_scanner/recos/millipede_original.py

```diff
@@ -43,7 +43,7 @@ class MillipedeOriginal(RecoInterface):
     # At HESE energies, deposited light is dominated by the stochastic losses
     # (muon part emits so little light in comparison)
     # This is why we can use cascade tables
-    _splinedir = os.path.expandvars("$I3_TESTDATA/photospline")
+    _splinedir = os.path.expandvars("$I3_DATA/photon-tables/splines")
```

Comment:
I'm a bit lost. How does this exist in CI if we are no longer bringing in the spline tables with the image build?

Reply:
The `prod` icetray image (the base image for this) installs a single spline table here, which just so happens to be the one we need for the original millipede.  Thus, CI works.